tests/utils.py

```diff
@@ -175,11 +179,10 @@ def upload_file(file_path):
     return s3.http_url(basename)
 
 
-def submit_job(app_url, app_args, app_name="/spark", args=[]):
+def submit_job(app_url, app_args, app_name=SPARK_APP_NAME, args=[]):
     if is_strict():
-        args += ["--conf", 'spark.mesos.driverEnv.MESOS_MODULES=file:///opt/mesosphere/etc/mesos-scheduler-modules/dcos_authenticatee_module.json']
-        args += ["--conf", 'spark.mesos.driverEnv.MESOS_AUTHENTICATEE=com_mesosphere_dcos_ClassicRPCAuthenticatee']
-        args += ["--conf", 'spark.mesos.principal=service-acct']
+        args += ["--conf", 'spark.mesos.containerizer=mesos']
```

Comment:
is MESOS containerizer required for `strict`? (wasn't in the past)

Reply:
Let me double-check ... I don't remember why I added it.